.github/workflows/ios-build-test-fabric.yml

```diff
@@ -39,6 +39,9 @@ jobs:
         working-directory: ${{ env.WORKING_DIRECTORY }}
         run: yarn
 
+      - name: Install Ruby gems
+        run: gem install cocoapods -v 1.15.2
+
```

Comment:
Isn't it enough to bump `cocoapods` dependeny in `Gemfile` of each application?

Reply:
I've tried to just use the range, but unfortunately Github Actions were still failing in my case